packages/gravity-ui-web/package.json

```diff
@@ -29,7 +29,7 @@
     "build": "gulp",
     "styleguide": "gulp styleguide",
     "start": "gulp serve",
-    "test": "npm run lint && bundlesize",
+    "test": "npm run lint && npm run build && bundlesize",
```

Comment:
Calling using the test phase was intentional so the build would fail and give feedback from Travis to Github.

Reply:
I just meant it might be nice to be able to call it separately as well as in the test phase. So something like:

```
  "start": "gulp serve",
  "bundlesize": "npm run build && bundlesize",
  "test": "npm run lint && npm run bundlesize",
```

but this is not something that would prevent merging, just a comment.